raysort/sort_utils.py

```diff
@@ -302,13 +302,13 @@ def calculate_boundaries(samples, n, bytes_for_bounds=8):
 def _run_valsort(argstr: str) -> str:
     proc = subprocess.run(
         f"{constants.VALSORT_PATH} {argstr}",
-        check=True,
+        check=False,
```

Comment:
Revert this before merging the PR.

Reply:
actually this is a fix. if you have `check=True`, then the subprocess.run function itself will raise an error. However, this isn't the behavior we want since in the next couple lines we check the proc status and do some additional processing before raising an error if we encountered one.

Reply:
I see. Add a comment here to explain?